src/main/scala/net/snowflake/spark/snowflake/SnowflakeRelation.scala

```diff
@@ -186,12 +191,7 @@ private[snowflake] case class SnowflakeRelation(
     val resultSetSerializables: Array[SnowflakeResultSetSerializable] =
       JavaConversions.asScalaBuffer(resultSet.asInstanceOf[SnowflakeResultSet]
         .getResultSetSerializables(params.expectedPartitionSize)).toArray
-
-    resultSetSerializables.zipWithIndex.map {
-      case (resultSet, index) => {
-        log.debug("partition  " + index + ": " + resultSet.toString())
-      }
-    }
+    printStatForSnowflakeResultSetRDD(resultSetSerializables)
```

Comment:
Is it going to download result to master? or just meta

Reply:
It's only metadata. This function only collects statistic from the metadata and print it in log.